firebase-vscode/package.json

```diff
@@ -94,7 +94,8 @@
     "build": "npm run copyfiles && webpack --config webpack.prod.js --devtool hidden-source-map",
     "build:extension": "webpack --config webpack.prod.js --config-name extension",
     "build:sidebar": "npm run copyfiles && webpack --config webpack.prod.js --config-name sidebar",
-    "pretest": "npm run dev ; npm run lint && tsc -p src/test/tsconfig.test.json",
+    "test-compile": "npm run copyfiles && webpack --config src/test/webpack.test.js",
```

Comment:
should prepublish or package be running tests?

Reply:
I think it can be added later when the tests do something. My preference would be I would want an overarching release process that runs each of the steps (test, build, package) separately, for more fine grained control.

Reply:
+1 to keeping each script to a single resposibility